test/fixtures/mson-multiline-description.json

```diff
@@ -0,0 +1,86 @@
+{
+  "_version": "3.0",
+  "metadata": [],
+  "name": "<API name>",
+  "description": "<API description>\n\n",
+  "element": "category",
+  "resourceGroups": [],
+  "content": [
+    {
+      "element": "category",
+      "content": [
+        {
+          "element": "object",
+          "meta": {
+            "id": "User"
+          },
+          "content": [
+            {
+              "element": "member",
+              "meta": {
+                "description": "An additional\nmulti-line description.\n\n- here\n\n- there\n"
+              },
+              "content": {
+                "key": {
+                  "element": "string",
+                  "content": "name"
+                },
+                "value": {
+                  "element": "string",
+                  "content": "Andrew"
+                }
+              }
+            },
+            {
+              "element": "member",
+              "meta": {
+                "description": "address description\nmultilined object\ndescription\n"
```

Comment:
@zdne @pksunkara what about some kind of separator?
like signature in mail or something simillar 
```
Short desctiption there
 --
Multilined description continue...
```